tests/dialects/test_bigquery.py

```diff
@@ -1419,7 +1419,7 @@ def test_bigquery(self):
             "SELECT GENERATE_DATE_ARRAY('2016-10-05', '2016-10-08')",
             write={
                 "duckdb": "SELECT CAST(GENERATE_SERIES(CAST('2016-10-05' AS DATE), CAST('2016-10-08' AS DATE), INTERVAL 1 DAY) AS DATE[])",
-                "bigquery": "SELECT GENERATE_DATE_ARRAY('2016-10-05', '2016-10-08')",
+                "bigquery": "SELECT GENERATE_DATE_ARRAY('2016-10-05', '2016-10-08', INTERVAL 1 DAY)",
```

Comment:
I think this actually makes it a breaking change, because the AST & its SQL output are different, let's make sure to mark as such before merging.